pkg/kubectl/scale.go

```diff
@@ -180,11 +197,63 @@ func (scaler *ReplicationControllerScaler) Scale(namespace, name string, newSize
 	return nil
 }
 
-// ScalerClient abstracts access to ReplicationControllers.
+// ScaleSimple is responsible for updating job's parallelism.
+func (scaler *JobScaler) ScaleSimple(namespace, name string, preconditions *ScalePrecondition, newSize uint) (string, error) {
```

Comment:
It's my first time in this code.  Can you add some godoc on the `ScaleSimple` method in the `Scaler` interface to explain what the return string is?